test/route.js

```diff
@@ -716,27 +716,6 @@ describe('Router', function () {
             .expect(200, { foo: 'fizz', bar: 'buzz' }, done)
         })
 
-        it('should work following a partial capture group', function (done) {
```

Comment:
While looking at the `express` tests I think we might be able to bring this one back in a different form. The interesting part is that the optional group is no longer captured because we dropped support for unnamed params. @blakeembrey just want to confirm this is our expected behavior.

This is the test as written in `express`:

```
    it('should work following a partial capture group', function(done){
      var app = express();
      var cb = after(2, done);

      app.get('/user{s}/:user/:op', function(req, res){
        res.end(req.params.op + 'ing ' + req.params.user + (req.url.startsWith('/users') ? ' (old)' : ''));
      });

      request(app)
      .get('/user/tj/edit')
      .expect('editing tj', cb);

      request(app)
      .get('/users/tj/edit')
      .expect('editing tj (old)', cb);
    })
```

Reply:
This is correct, there's no regex or unnamed group features right now. I think we could bring this back given time but right now it isn't possible.